packages/extension-chakra-storefront/src/components/product-view/index.jsx

```diff
@@ -40,11 +40,9 @@ const ProductViewHeader = ({
 }) => {
     return (
         <VStack mr={4} gap={2} align="flex-start" marginBottom={[4, 4, 4, 0, 0]}>
-            {category && (
-                <Skeleton loading={!category} minWidth={64}>
-                    <Breadcrumb categories={category} />
-                </Skeleton>
-            )}
+            <Skeleton loading={!category} minWidth={64}>
+                {category && <Breadcrumb categories={category} />}
+            </Skeleton>
```

Comment:
Bonus: like what happened to the rest of the Skeletons on this page, I'm applying the same treatment here to this Skeleton. If a check is needed (in this case, checking `category`), let's make sure it's done within the Skeleton.

Reply:
You might have opened a can of worms here. But do we use this component in placed where there isn't category data ever. Would this change make it so that a skeleton is showing when we don't want it to?

But this begs the question, did the old logic make any sense at all?

Reply:
I've reverted the change for now :)